internal/core/input.rs

```diff
@@ -131,6 +131,28 @@ pub mod key_codes {
     macro_rules! declare_consts_for_special_keys {
        ($($char:literal # $name:ident # $($_qt:ident)|* # $($_winit:ident)|* ;)*) => {
             $(pub const $name : char = $char;)*
+
+            /// A keyboard key 
+            pub enum Key {  
+                /// The given key is unknown.
+                Unknown,
```

Comment:
Why do we need unknown? For default? But do we need default?
It should be non exhaustive thought

Reply:
That's what I saw on different sources describing Keys (Unknown/Undefined). But I think I our case it's not mandatory.